.github/workflows/deploy-2-start.yml

```diff
@@ -64,11 +65,14 @@ jobs:
             ${{ secrets.HOST }}
             ${{ secrets.HOST_DATA }}
 
-      - name: Prerelease Modulefile Name
+      - name: Prerelease spack.yaml Modifications
         if: inputs.type == 'prerelease'
         # Modifies the name of the prerelease modulefile to the
-        # `pr<number>-<commit>` style. For example, `access-om3/pr12-2`
+        # `pr<number>-<commit>` style. For example, `access-om3/pr12-2`.
+        # Also removes the `@git.VERSION` specifier for Prereleases so
+        # we don't have to shift tags around.
```

Comment:
What does it replace the `@git.VERSION` with?

Reply:
With nothing, so it's just `- access-om2-bgc` for example. The version is implicit.

Reply:
So how do we make sure we're building the PR version?

Reply:
The spack.yaml itself is the PR version - it is the file that is deployed

Reply:
Happy days!